app/main/views/services.py

```diff
@@ -459,6 +459,35 @@ def complete_draft_service(framework_slug, lot_slug, service_id):
                                 lot=lot_slug))
 
 
+@main.route('/frameworks/<framework_slug>/submissions/<lot_slug>/<service_id>/delete', methods=['GET'])
+@login_required
+@EnsureApplicationCompanyDetailsHaveBeenConfirmed(data_api_client)
+@return_404_if_applications_closed(lambda: data_api_client)
+def delete_draft_service_warning(framework_slug, lot_slug, service_id):
```

Comment:
Can we be clearer about what this method is doing? I initially misread this as 'delete [the] (draft service warning)' instead of '(delete draft service) warning'.

(ditto for the other 3 methods)